core/src/test/java/com/sequenceiq/cloudbreak/TestUtil.java

```diff
@@ -0,0 +1,127 @@
+package com.sequenceiq.cloudbreak;
```

Comment:
Please make this a "real" utility class (with private constructor, static methods) rather than a base class for test classes.